packages/components/atoms/f-form-field/src/components/FormDropdown.vue

```diff
@@ -1,6 +1,7 @@
 <template>
     <div
         :class="$style['c-formDropdown']"
+        :disabled="attributes.disabled"
```

Comment:
Probably worth having a hangout about this one as think getting wires crossed and will be easier to go through it that way. Let me know when you're free.